polar-core/src/partial/partial.rs

```diff
@@ -143,7 +143,10 @@ impl Partial {
 
         let op = Operation {
             operator,
-            args: vec![self.variable_term(), other],
+            args: match operand {
```

Comment:
Would it be beneficial to normalize operations so that `this` is always on the LHS?

That could eliminate the need for branches for both in all adapters.

Reply:
Done in e8654d2. Good suggestion!